canonical-web-and-design/ubuntu.com
# Add some redirects to '/cloud'

## Done

Add the following redirects
- [ ] /private-cloud, /hybrid-cloud, /multi-cloud, /edge-cloud redirecting to /cloud
- [ ] /openstack/support redirecting to /advantage

## QA

- Wait for the demo 
- Check the redirects above work

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3871
